src/legacy/core_plugins/kibana/public/dashboard/panel/panel_utils.ts

```diff
@@ -17,25 +17,37 @@
  * under the License.
  */
 
-import _ from 'lodash';
 import { i18n } from '@kbn/i18n';
-import { DEFAULT_PANEL_WIDTH, DEFAULT_PANEL_HEIGHT } from '../dashboard_constants';
+import _ from 'lodash';
 import chrome from 'ui/chrome';
+import { DEFAULT_PANEL_HEIGHT, DEFAULT_PANEL_WIDTH } from '../dashboard_constants';
+import { GridData, PanelState } from '../selectors';
 
 const PANEL_HEIGHT_SCALE_FACTOR = 5;
 const PANEL_HEIGHT_SCALE_FACTOR_WITH_MARGINS = 4;
 const PANEL_WIDTH_SCALE_FACTOR = 4;
 
 export class PanelUtils {
-
   // 6.1 switched from gridster to react grid. React grid uses different variables for tracking layout
-  static convertPanelDataPre_6_1(panel) { // eslint-disable-line camelcase
+  public static convertPanelDataPre_6_1(panel: {
+    panelIndex: any; // earlier versions allowed panelIndex to be a number or a string
+    gridData: GridData;
+    col: number;
+    row: number;
+    size_x: number;
+    size_y: number;
+    version: string;
+  }): Partial<PanelState> {
+    // eslint-disable-line camelcase
```

Comment:
Is this line necessary?  (perhaps it's a carryover from the original function definition?)

Reply:
yep line doesn't seem necessary - removed it

Reply:
fc7f3db